### PR TITLE
Handle data with Publish and Synchronization Last updated

### PR DESCRIPTION
**Pull request by bell87**
## PR Description
Added detail to the 'Client' step of this tutorial so HelloWorld message data can be returned from redis

## Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

**Comment by bell87** [comment]
The 'checks' seems to fail with the external link check step?
```
Run bundle exec rake check_paas_plus
  bundle exec rake check_paas_plus
  shell: /usr/bin/bash -e {0}
NOTE: nokogumbo: Using Nokogiri::HTML5 provided by Nokogiri. See https://github.com/sparklemotion/nokogiri/issues/2205 for more information.
Running ["HtmlCheck", "ImageCheck", "LinkCheck", "ScriptCheck"] on ["./_site"] on *.html... 


Checking 30 external links...
- ./_site/docs/paas-plus/dev/platform-as-a-service-plus.html
Ran on 20716 files!
  *  External link https://buddy.works/docs failed: got a time out (response code 0)



HTML-Proofer found 1 failure!
Error: Process completed with exit code 1.
```

File: docs/scos/dev/back-end-development/data-manipulation/data-publishing/handle-data-with-publish-and-synchronization.md [hunk 1]
```diff
@@ -706,7 +716,88 @@ class HelloWorldStorageFacade extends AbstractFacade implements HelloWorldStorag
...
+
+namespace Pyz\Zed\HelloWorldStorage\Business;
+
+use Pyz\Zed\HelloWorldStorage\Business\Deleter\HelloWorldStorageDeleter;
+use Pyz\Zed\HelloWorldStorage\Business\Deleter\HelloWorldStorageDeleterInterface;
+use Pyz\Zed\HelloWorldStorage\Business\Writer\HelloWorldStorageWriter;
+use Pyz\Zed\HelloWorldStorage\Business\Writer\HelloWorldStorageWriterInterface;
+use Spryker\Zed\Kernel\Business\AbstractBusinessFactory;
+
+
```
[the remark this answers is not in the document]
Comment (dmiseev) [reply docs/scos/dev/back-end-development/data-manipulation/data-publishing/handle-data-with-publish-and-synchronization.md:734]: Please check other places.

File: docs/scos/dev/back-end-development/data-manipulation/data-publishing/handle-data-with-publish-and-synchronization.md [hunk 2]
```diff
@@ -933,34 +1024,237 @@ This section describes how to read the data from Redis.
...
+```xml
+<?xml version="1.0"?>
+<transfers xmlns="spryker:transfer-01"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="spryker:transfer-01 http://static.spryker.com/transfer-01.xsd">
+    <transfer name="HelloWorldStorage">
+        <property name="name" type="string" />
+        <property name="message" type="string" />
+        <property name="id" type="integer" />
```
Comment (dmiseev) [inline docs/scos/dev/back-end-development/data-manipulation/data-publishing/handle-data-with-publish-and-synchronization.md:1191]: type="int"

**Comment by lenadoc** [comment]
Hi Andrew! Thank you for your contribution! It is very much appreciated.

Best regards,
The Spryker Docs team

**Pull request merged**